playground/mocks/data/idp/idx/error-pre-versioning-ff-session-expired.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version":"1.0.0",
+  "messages":{
+    "type":"array",
+    "value":[
+      {
+        "message":"The session has expired.",
```

Comment:
we have `error-session-expired.json` already. curious what's the difference?

Reply:
That was added by Pradeep to deal with polling.  I don't know much about that one, but when I compared that JSON with the responses I was getting from the server for this case, the non-polling responses did not contain the stateHandle, yet `error-session-expired.json` does, so I created this second more explicit mock json.